membership-attribute-service/app/controllers/ExistingPaymentOptionsController.scala

```diff
@@ -72,9 +70,10 @@ class ExistingPaymentOptionsController(commonActions: CommonActions, override va
   def cardThatWontBeExpiredOnFirstTransaction(cardDetails: PaymentCardDetails) =
     new LocalDate(cardDetails.expiryYear, cardDetails.expiryMonth, 1).isAfter(now.plusMonths(1))
 
-  def existingPaymentOptions(currencyFilter: Option[String]) = BackendFromCookieAction.async { implicit request =>
+  def existingPaymentOptions(currencyFilter: Option[String]) = AuthAndBackendViaIdapiAction(ContinueRegardlessOfSignInRecency).async { implicit request =>
     implicit val tp = request.touchpoint
-    val maybeUserId = authenticationService.userId
+    val maybeUserId = request.redirectAdvice.userId
+    val isFreshlySignedIn = request.redirectAdvice.signInStatus == SignedInRecently
```

Comment:
Variable name `isFreshlySignedIn` could be changed to `isSignedInRecently` to make it consistent with the check we're doing to derive the variable?

Reply:
good point - done in https://github.com/guardian/members-data-api/pull/384/commits/1ff58275001f05802ed8dfe034a0e36e40cbbafe